Charts.xcodeproj/project.pbxproj

```diff
@@ -1307,6 +1343,25 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
```

Comment:
@jjatie could you give some context what are these changes about?

Reply:
Again, autogenned by Xcode when importing a Swift package to an xcproj. Required so that Cocoapods and Carthage build with our SPM dependencies.

This reminds me I forgot to update the Package.swift manifest for SPM build 🤦‍♂️